### PR TITLE
Api key changes

### PR DESCRIPTION
**Pull request by a-ramit**

### CODE REVIEW

**Comment by a-ramit** [comment]
These are the changes as discussed with Airtel Team. Please review.

**Comment by bigludo7** [comment]
Hi
Adding @monamok as reviewer as well.
Thanks

**Comment by monamok** [comment]
Hello @a-ramit,
I would like to point out here that the most important thing about this API is that for verifying the number, authentication must be via network (IP address). That is why a 3-legged access token is required. You can also see that there is a specific error 403 just designed for it, in case another authentication method was used:
{
    "code": "NUMBER_VERIFICATION.USER_NOT_AUTHENTICATED_BY_IP",
    "message": "Client must authenticate via IP to use this service"
}

Using API key there is no way to guarantee that the phone number is the same one used in the device and there is no information bound to the apikey regarding the phone_number.

File: code/API_definitions/CAMARA/number_verification.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ info:
     It enables a Service Provider (SP) to verify the phone number of the mobile device being used to access their service where the mobile device is accessing the *service provider* over a mobile network (WiFi connections are out of this API scope) either by getting the comparison result or receiving the phone number of the device that it is used, so they can verify it themselves.
 
     # Resources and Operations overview
-    This API currently provides two endpoints where both require a **3-legged token** and authentication via **mobile network** (excluding for example by SMS/OTP or user/password as an authentication method):
+    This API currently provides two endpoints where both require either a **3-legged token** or **API Key** and authentication via **mobile network** (excluding for example by SMS/OTP or user/password as an authentication method):
```
Comment (DT-DawidWroblewski) [inline code/API_definitions/CAMARA/number_verification.yaml:14]: @a-ramit can we have a call to go through the flow? Can you open an issue inside Commonalities WG to discuss API-key topic.

**Comment by bigludo7** [comment]
No update from last 2 months. Please open another issue if need to be discuss.

**Pull request closed without merging**